agenthub/coact_agent/meta/planner_prompt.py

```diff
@@ -0,0 +1,33 @@
+PLANNER_CONSTRUCT_PLAN = """Your role is to construct a multi-stage global plan, providing separate subtask descriptions and expected states for each phase. Ensure that your plan is comprehensive and covers all aspects of the task. Use the following format:
+
+**Subtask 1:**
+- **Subtask**: Describe the first subtask you are about to plan.
+- **Expected State**: Define the expected state of the task after completing the first subtask.
+
+**Subtask 2:**
+- **Subtask**: Describe the second subtask you are about to plan.
+- **Expected State**: Define the expected state of the task after completing the second subtask.
+
+...
+
+**Subtask n:**
+- **Subtask**: Describe the nth subtask you are about to plan.
+- **Expected State**: Define the expected state of the task after completing the nth subtask.
+"""
```

Comment:
Are there defined or "preferred" states (that should be stated here)? Seems a bit open to discussion for the LLM what that means.

Reply:
I have defined a more detailed structure in the system prompt. Can you have a look at it? (I don't use these "meta prompts" yet, just placeholder for now)